.travis.yml

```diff
@@ -16,12 +16,9 @@
 # under the License.
 
 language: java
-cache:
-  directories:
-    - $HOME/.m2
 before_install:
-  - echo "MAVEN_OPTS='-Xms512m -Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m'" > ~/.mavenrc
+  - export MAVEN_SKIP_RC="true"
```

Comment:
Do you have evidence that this line is required? If so, can you please add a comment about it?